zabbixprovisioner/provisioner/provisioner.go

```diff
@@ -205,6 +205,11 @@ func (p *Provisioner) LoadRulesFromPrometheus(hostConfig HostConfig) error {
 			newTrigger.Trigger.Expression = fmt.Sprintf("{%s:%s.nodata(%s)}", newHost.Name, key, delay)
 		}
 
+		// Add the special "Prefix" to trigger if requested
+		if prefix, ok := rule.Annotations["zabbix_trigger_prefix"]; ok {
+			newTrigger.Trigger.Description = fmt.Sprintf("%s%s", prefix, newTrigger.Trigger.Description)
```

Comment:
Why? Prefix = Description? Maybe we can make this configureable ? 

Also this feels like a config for provisioner binary not a config for each rule. What do you think?

Reply:
it could be a good solution if every rule has same "prefix tag" but it 's not, each rule can have a different "prefix tag"

Reply:
the maybe we can map prefix tag to prefix annotation instead?